app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
The `collection_of_coins` variable is categorized as mutable, but sets in Python are mutable. Ensure that this categorization aligns with the intended use of the variable. If `collection_of_coins` is intended to be mutable, then the categorization is correct. Otherwise, consider revising it.